src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -87,6 +87,34 @@ Blank()
 >> Sum(Table(ParseJSON("[1, 2, 3, 4, 5]")), Value(Value))
 15
 
+>> ForAll(ParseJSON("[5, 6, 7]"), Value(ThisItem))
+[5,6,7]
+
+>> ForAll(ParseJSON("null"), Value(ThisItem))
+Blank()
+
+// We currently do not support null in any of the constructor functions, so this is an error
```

Comment:
>do not support null in any of the constructor functions [](http://example.com/codeflow?start=16&length=55)

I thought we did support null in the .ctor functions - see tests on lines 20, 48, 66, and many others.

Reply:
Fixed